docs/requirements.md

```diff
@@ -0,0 +1,49 @@
+At its heart ServiceControl and ServicePulse provide operational support for systems built on the Particular Service Platform. They do this by:
+
+- monitoring system health
+- diagnosing and treating errors
+
+In this context, system refers to NServiceBus endpoints and communication with third-party services.
+
+ServiceControl should provide the following features:
```

Comment:
For the final version I'd like to get rid of "ServiceControl" term because it implies monolithic design whereas I believe the features listed below can be fulfilled by a set of fairly independent components.